modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: CONCEPT
+
+[id="relocation-of-openshift-jenkins-images_{context}"]
+= Relocation of OpenShift Jenkins images
+
+.What stays the same with the OpenShift Jenkins images?
+
+* The Cluster Samples Operator manages the `ImageStream` object and `Template` objects for operating the OpenShift Jenkins images.
+* By default, the Jenkins `DeploymentConfig` object from the Jenkins pod template triggers a redeployment when the Jenkins image changes. By default, this image is referenced by the `jenkins:2` image stream tag of Jenkins image stream in the `openshift` namespace in the `ImageStream` YAML file in the Samples Operator payload.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the deprecated `maven` and `nodejs` pod templates are still in the default image configuration.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams still exist in your cluster. To maintain these image streams, see the "So what happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?" section below.
+
+.What changes in the support matrix of the OpenShift Jenkins image?
+
+Each new image in the `ocp-tools-4` repository in the `registry.redhat.io` registry supports multiple versions of {product-title}. When Red Hat updates one of these new images, it is available for all versions simultaneously. This availability is ideal when Red Hat updates an image in response to a security advisory.
```

Comment:
We should mention that this will apply to 4.11 and upward if a first step.
And then, this will be from 4.9 to upward.

Reply:
Updated. Thanks.